tmdb/parsers/shows.py

```diff
@@ -70,6 +71,17 @@ def _get_next_episode_date(self, data: dict) -> Union[datetime.date, None]:
         next_episode: dict = data['next_episode_to_air'] or {}
         return self._parse_date(next_episode.get('air_date'))
 
+    def _get_list_seasons(self, data: dict) -> List[Season]:
```

Comment:
Perhaps for simpler semantics we could call a `List[Season]` to be `seasons` and a raw `List[dict]` to be `raw_seasons` or something…

In fact, the `list_seasons` (or even `seasons_list`) denomination is not Pythonic because it binds the type to the variable name (but the type is already shown using type annotations).

Reply:
I dont get it